### DISCUSSION TITLE
Agent DVR addon for Home Assistant

### DISCUSSION THREAD

**Discussion by DivanX10** [doitandbedone/ispyagentdvr-docker#471] [General]
I want to offer you an idea. This is to create an Agent DVR addon for a home assistant. This will make AgentDVR more friendly and as user-friendly as possible. The home assistant works on different machines and not everyone has servers on hypervisors and not everyone has Synology, many use Raspbian for a home assistant. Also, not everyone is friends with linux and cannot install AgentDVR according to this [instruction](https://www.ispyconnect.com/download.aspx ). Also to install AgentDVR via [portainer](https://github.com/hassio-addons/addon-portainer ), which is built into the home assistant, you need to be able to lift containers, which is also not everyone can do and therefore not everyone can install AgentDVR, although they really wanted to use AgentDVR.  AgentDVR is a great project, but due to the complexity of installation or requiring a separate server, AgentDVR is not used by many

Thank you so much for such a wonderful project as AgentDVR 😀

**Comment by wabragg** [comment]
Odd, when I brought AgentDVR up my Home Assistant found it and added a panel for it in the Lovelace interface immediately.  I'm running 3.6.2 on a Synology NAS in Docker.  HomeAssistant is on the same NAS with version 2021.9.

**Comment by DivanX10** [comment]
> Odd, when I brought AgentDVR up my Home Assistant found it and added a panel for it in the Lovelace interface immediately. I'm running 3.6.2 on a Synology NAS in Docker. HomeAssistant is on the same NAS with version 2021.9.

You write about AgentDVR integration, and I write about the AgentDVR addon. These are two completely different things. The AgentDVR addon will allow you not to deploy AgentDVR on an external server, but will be installed automatically in HomeAssistant, as other addons are installed in HomeAssistant

**Comment by wabragg** [comment]
Thanks.  I've learned something new then.
